pkg/kv/kvpb/api.proto

```diff
@@ -1996,6 +1996,45 @@ message AddSSTableRequest {
   // also find and return the key at which the span after the added file span
   // is likely non-empty. See AddSSTableResponse.FollowingLikelyNonEmptySpanStart.
   bool return_following_likely_non_empty_span_start = 9;
+
+  // RemoteFile indicates that the span indicated by the request start and end 
```

Comment:
should we package these new proto fields into a single `remoteIngestion` message to prevent bloat of the addsstable request proto? I don't feel too strongly about this.

Reply:
sounds good.